sandbox/snips/learn-multi.py

```diff
@@ -0,0 +1,40 @@
+import nengo
```

Comment:
Is this file supposed to be in here or included accidentally?

Reply:
Oops, sorry -- that was an accident.